app/views/practices/form/metrics.html.erb

```diff
@@ -0,0 +1,245 @@
+  <%= javascript_include_tag 'https://www.gstatic.com/charts/loader.js' %>
+  <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
+    <%= render partial: 'practices/form/metrics', formats: [:js] %>
+  <% end %>
+<% end %>
+
+<%= render 'practice_editor_sidenav' %>
+<div class="grid-container position-relative">
+  <div class="grid-row grid-gap practice-editor-min-height">
+    <div class="desktop:grid-col-3 grid-col-auto z-bottom">&nbsp;</div>
+
+    <div id="metrics" class="desktop:grid-col-9 grid-col-12 p0">
+      <div class="margin-bottom-4">
+      <%= render partial: "shared/messages", locals: {small_text: false} %>
+    </div>
+      <section class="usa-section padding-top-0 p0-bottom contact">
+        <p style="font-weight: bold; font-size: 40px" class="margin-bottom-3">
+          Metrics
+          <span style="float: right; padding-right: 30px">
+            <a class="dm-icon-link text-no-underline font-sans-xs text-primary desktop:grid-col-3 grid-col-6 margin-bottom-0"
+               onclick="window.print(); return false;">
+              <i class="fas fa-print margin-right-05"></i>
+              <span>Print</span>
+            </a>
+          </span>
+        </p>
+        <div class="margin-bottom-10">
+          View your metrics for
+          <span class="display-inline-block">
+            <select class="usa-select metrics-select" name="metrics_duration" id="metrics_duration">
+              <option id="metrics_30_day_value" value="30" <%= 'selected' if @duration.to_i == 30 %>>the last 30 days</option>
+              <option id="metrics_all_time_value" value="99999" <%= 'selected' if @duration.to_i != 30 %>>all time</option>
+            </select>
+          </span>
+        </div>
+          <div class="grid-col-8 margin-bottom-10"  style="text-align: center; width: 100%" >
+          <table style="width: 80%; display: inline-table">
+            <th style="font-size: 13px">PAGE VIEWS</th>
+            <th style="font-size: 13px">UNIQUE VISITORS</th>
+            <th style="font-size: 13px">BOOKMARKS</th>
+            <th style="font-size: 13px">ADOPTIONS</th>
+            <tr>
+              <td style="font-size: 32px"><%= @page_views_for_practice_count %></td>
+              <td style="font-size: 32px"><%= @unique_visitors_for_practice_count %></td>
+              <td style="font-size: 32px"><%= @bookmarks_by_practice %></td>
+              <td style="font-size: 32px"><%= @adoptions_by_practice %></td>
+            </tr>
+          </table>
+          </div>
+        <!--  CHARTS ------------------------------------------------------------>
+
+        <div class="border bg-white padding-2 margin-bottom-10 radius-md padding-bottom-0">
+          <div style="width: 726px; height: 400px" id="chart_div"></div>
+         </div>
+
+
+<!-- ADOPTIONS ------------------------------------------------------------->
+        <div class="margin-bottom-3">
```

Comment:
for the table rows and headers for the adoptions and leaderboard section, there needs to be more padding around the text

Reply:
Done